cms/lib/xblock/tagging/tagging.py

```diff
@@ -90,3 +90,14 @@ def save_tags(self, request=None, suffix=None):  # pylint: disable=unused-argume
             return Response("Invalid 'tag' parameter", status=400)
 
         return Response()
+
+    def get_event_context(self, event_type, event):  # pylint: disable=unused-argument
+        """
+        This method return data that should be associated with the "check_problem" event
+        """
+        if event_type != "problem_check":
+            return
```

Comment:
Nit: this should be `return None`, since the other spots in the method return an object. (Use `return` if the method isn't ever going to return anything).